### DISCUSSION TITLE
feat(DateTimeRange): Expose properties to public

### DISCUSSION THREAD

**Discussion by eramosr16** [dotnetcore/BootstrapBlazor#3405] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

I'm trying to:

-  Hide/Show the Right/Left buttons on the DateTimeRange component header component but is not possible to do since the properties are hard coded inside the component.

- Set DateTimeRange to be displayed as a single DateTime component, right now it is displayed as two DateTime components but I wanna be able to display it as a single DateTime. 

### Describe the solution you'd like

Expose the:
- ShowLeftButtons and ShowRightButtons as properties to DateTimeRangeComponent.

- Add option to DisplayComponent as a Single/Double DateTimePicker.

- Click and Drag StartDate/EndDate inside the component to modify Ranges selected.

**Comment by bb-auto** [comment]
@eramosr16 Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@eramosr16 

1. Hide/Show the Right/Left buttons on the DateTimeRange component header component but is not possible to do since the properties are hard coded inside the component.

Similar to #3393 it's by design

2. Set DateTimeRange to be displayed as a single DateTime component, right now it is displayed as two DateTime components but I wanna be able to display it as a single DateTime.

single DateTime component is `DateTimePicker`
double DateTime component is `DateTimeRange`

The `DateTimeRange` component currently plans to change to better support time
